rails/benchmarks/activemodel_views.rb

```diff
@@ -37,7 +39,7 @@ def user_factory
   { email: Faker::Internet.email }
 end
 
-def render_views
+m = Benchmark.rails(100, "actionview/render_activemodels") do
```

Comment:
@kirs Just a thought. Since this script is displayed on the site, Would it be better if we use `kwargs` here? 

```Benchmark.rails(n: 1000, script_name: "actionview/render_activemodels", gc: false)```